frontend/app/services/personal-information-service.server.ts

```diff
@@ -185,5 +151,48 @@ function createPersonalInformationService() {
     throw new Error(`Failed to fetch data. address: ${response.status}, Status Text: ${response.statusText}`);
   }
 
+  function toPersonalInformation(personalnformationDto: PersonalInformationApiResponse): PersonalInfo {
```

Comment:
It might be confusing for devs that this is named `personalnformationDto` of type `PersonalInformationApiResponse`, but there is a `personalInfoDtoSchema` above, that represents our application domain model.

```suggestion
  function toPersonalInformation(personalInformationApiResponse: PersonalInformationApiResponse): PersonalInfo {
```

Reply:
Changed:  29206a2f127b07482bac775716bc539e0a828f36